static/templates/composer.tpl

```diff
@@ -1,4 +1,4 @@
-<div component="composer" class="composer<!-- IF resizable --> resizable<!-- ENDIF resizable -->">
+<div component="composer" class="composer<!-- IF resizable --> resizable<!-- ENDIF resizable --><!-- IF isTopicOrMain --> topic-or-main<!-- ENDIF isTopicOrMain -->">
```

Comment:
The point of this change is to hide the title in zen mode when someone is simply replying to provide for even more space.